pylon-core/src/main/kotlin/io/github/pylonmc/pylon/core/item/base/PylonCooldownable.kt

```diff
@@ -3,7 +3,8 @@ package io.github.pylonmc.pylon.core.item.base
 import org.jetbrains.annotations.ApiStatus
 
 /**
- * Implemented by interfaces affected by cooldowns
+ * Implemented by interfaces that may be affected by cooldowns. You should not
+ * implement this interface on an item yourself.
```

Comment:
```suggestion
 * Implemented by interfaces that may be affected by cooldowns.
```
It's `sealed` so they can't implement it anyway